corehq/messaging/tasks.py

```diff
@@ -191,11 +191,12 @@ def run_messaging_rule_for_shard(domain, rule_id, db_alias):
 
     chunk_size = getattr(settings, 'MESSAGING_RULE_CASE_CHUNK_SIZE', 100)
     progress_helper = MessagingRuleProgressHelper(rule_id)
-    for case_id_chunk in chunked(paginated_case_ids(domain, rule.case_type, db_alias), chunk_size):
-        sync_case_chunk_for_messaging_rule.delay(domain, case_id_chunk, rule_id)
-        progress_helper.increase_total_case_count(len(case_id_chunk))
-        if progress_helper.is_canceled():
-            break
+    if not progress_helper.is_canceled():
```

Comment:
Can we not simply return after this line instead of tabbing following lines?

Reply:
we would still need to update that the shard is complete which happens later.

Reply:
👍

Reply:
cool, we good?